runtime/parachains/src/inclusion.rs

```diff
@@ -320,17 +325,23 @@ impl<T: Config> Pallet<T> {
 			for (bit_idx, _) in
 				signed_bitfield.payload().0.iter().enumerate().filter(|(_, is_av)| **is_av)
 			{
-				let (_, pending_availability) = assigned_paras_record[bit_idx]
-					.as_mut()
-					.expect("validator bitfields checked not to contain bits corresponding to unoccupied cores; qed");
+				let pending_availability = if let Some((_, pending_availability)) =
+					assigned_paras_record[bit_idx].as_mut()
+				{
+					pending_availability
+				} else {
+					// only happens in case of unoccupied cores, which in turn happens in case
+					// of disputes.
```

Comment:
This comment gives the wrong impression that validators only act with good intentions. Can you extend it to mention the fact that malicious validators setting arbitrary bits will have a minimal effect?